crates/tap-agent/src/test.rs

```diff
@@ -260,6 +263,65 @@ pub fn create_rav(
     .unwrap()
 }
 
+pub trait CreateReceipt {
```

Comment:
So this is a static function effectively? I confused looking a trait implementation with no `self`.

Reply:
Yes, we don't need to get anything from Self. We just want different function implementations given the T being implemented